src/python/dbs/apis/dbsClient.py

```diff
@@ -10,6 +10,16 @@
 import sys
 import urllib.request, urllib.parse, urllib.error
 
+def headerHas(headers, content, mime):
+    """
+    Check mime content in HTTP headers which represented as string
```

Comment:
Given that this client will be used by many applications, and in case auto-documentation is implemented, I would suggest to expand on this docstring and also explain the arguments and their types and what is returned.

For instance, I believe `headers` is a plain multi line string, right?

Reply:
done